sql/ddl_api.sql

```diff
@@ -23,7 +23,7 @@
 -- replication_factor - (Optional) A value of 1 or greater makes this hypertable distributed
 -- data_nodes - (Optional) The specific data nodes to distribute this hypertable across
 CREATE OR REPLACE FUNCTION  create_hypertable(
-    main_table              REGCLASS,
+    hypertable              REGCLASS,
```

Comment:
This would be `relation` since there is no hypertable at this point.
```suggestion
    relation              REGCLASS,
```